ee-feature-pack/galleon-common/src/main/resources/layers/standalone/web-clustering/layer-spec.xml

```diff
@@ -15,5 +15,13 @@
              and undertow subsystem doesn't assume clustering, but the
              combination requires the clustering<->undertow integration package -->
         <package name="org.wildfly.clustering.web.undertow"/>
+        <!-- This layer also brings in Infinispan modules -->
+        <package name="org.infinispan.cdi.common"/>
+        <package name="org.infinispan.cdi.embedded"/>
+        <package name="org.infinispan.cdi.remote"/>
+        <package name="org.infinispan.counter"/>
+        <package name="org.infinispan.lock"/>
+        <package name="org.infinispan.query"/>
+        <package name="org.infinispan.query.core"/>
```

Comment:
Is it possible to instead define these package dependencies within the infinispan feature group?

Reply:
@pferraro Turns out it is actually possible! Reworked.